CONTRIBUTING.md

```
@@ -83,8 +83,11 @@ There is a great reference for commit messages in the
 IMPORTANT: you must sign-off your work using `git commit --signoff`. Follow the
 [Linux kernel developer's certificate of origin][linux-signoff] for more
 details. All contributions are made under the
-[BSD-3-Clause][https://spdx.org/licenses/BSD-3-Clause.html] license. Please use
-your real name and not a pseudonym. Here is an example:
+[BSD-3-Clause][https://spdx.org/licenses/BSD-3-Clause] license, except for
```

markdownlint-cli2 (0.22.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "https://spdx.org/licenses/bsd-3-clause"

(MD052, reference-links-images)